### DISCUSSION TITLE
Fire only burns for 30 s even though I want it to burn for much longer

### DISCUSSION THREAD

**Discussion by calania** [firemodels/cfast#2062] [Q&A] [answered]
Hello! I have been working on a project where I want to simulate a fire in a house consisting out of three rooms. I have added a t^2 fire to one room  with a time to peak and decay time of 1 s and a steady burning period if 8000 s. While testing I also made the peak HRR 10000 KW. My problem is that even though I have set the steady burning period to 8000s the fire only actually burns for roughly 30 s and after that the temperatures rapidly decreases when I run the simulation. I have been banging my head against the wall and can't figure out what I'm doing wrong. I have making a new file redoing everything but with the same results. Can anybody here see what's wrong?

<img width="1440" alt="Screenshot 2023-10-21 at 09 40 14" src="https://github.com/firemodels/cfast/assets/34073343/fd6453ca-cd94-422a-a4fb-aea801c444e5">

<img width="1440" alt="Screenshot 2023-10-21 at 09 40 52" src="https://github.com/firemodels/cfast/assets/34073343/13bc2d2a-7dec-47e1-8fc3-99ee07e92fcc">

<img width="1440" alt="Screenshot 2023-10-21 at 09 41 02" src="https://github.com/firemodels/cfast/assets/34073343/9ebe09f2-46b3-4c27-82f3-e367f8440017">

<img width="1440" alt="Screenshot 2023-10-21 at 09 41 29" src="https://github.com/firemodels/cfast/assets/34073343/76abc290-4d5b-4ca8-806a-29ba551d6c82">

**Comment by drjfloyd** [comment, marked as answer]
Fires require oxygen to burn. A 10 MW fire wil require ~0.75 kg O2/s to burn.  You have ~130 m^3 of compartment volume which means you start with ~37 kg of oxygen.  You have no vents to the exterior for more oxygen to enter.

**Comment by calania** [comment]
Well that's embarrassing. Don't know how I forgot about something as basic as the fire needing air. Adding some ventilation fixed it. Thanks for the help
